Added section for Wii U guide to switch between Nintendo Network and Pretendo Network profiles (#229)
PretendoNetwork/website: docs/en_US/install/wiiu.md

<img src="/assets/images/docs/install/wiiu/pnid/pretendo-eula.jpg" width="100%">

Finally, set up your PNID, email and password. Once all of this is done, you will be able to use Pretendo servers.

# Switching between Pretendo Network and Nintendo Network

Occasionally, you may want to switch between your Pretendo Network and your Nintendo Network profiles on your Wii U; fortunately, it is super simple to switch between the two.

While connected to Pretendo, in the HOME Menu, switch to your Nintendo Network profile. If prompted for a password, do not provide it (if you don't have auto sign-in enabled, you may not have the option to not provide it). You are now inside your Nintendo Network profile.

> On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:

Reviewer: Yeah but you're going to have to switch to your NNID to be able to connect to Nintendo Network

> On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:

Reviewer 2: The reboot that happens after disabling the patches does an actual console soft reboot, you get sent back to the profile selection screen. So selecting a profile does not do anything, as you will leave the profile the moment you enable/disable the patches

> On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:

Author: I don't remember being taken back to that screen personally but I have my Wii U set to my NNID as default

> On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:

Author: I don't really find it's a big issue anyways having that in but if you think that should be changed lmk

<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">

> Your menu might display additional plugins that aren't shown in this image; however, our focus is specifically on the 'Inkay' plugin, which patches requests intended for Nintendo Network and reroutes them to Pretendo Network instead. Once you've found the entry for Inkay, press `A`.

Reviewer 2: I'm gonna use this paragraph as an example but this comment goes for basically the whole page

While I do enjoy having a fun/"personal" vibe with our brand generally, when it comes to our official documentation I do prefer to keep it as short, concise, and professional as possible. That will likely make the documentation come off as "corpo", but there are tons of people who won't understand the instructions unless it's handed to them in the simplest way possible

Doing things this way avoids sections that get too wordy with no real benefit to the user, and gets directly to the point of the section

There's no hard set of rules or guidelines for this though. I kinda just judge if based off vibes. Like for this section I would suggest something like

```suggestion
You may have additional plugins in your menu. Navigate to the `Inkay` plugin and press `A` to enter it's settings.
```

This removes a good portion of fluff from this section, mostly related to the fact that it's already clear what Inkay does based on information earlier in the page. That doesn't need to be restated

Reviewer: Ok that's also on me. whoops

Reviewer 2: It's not just your changes I actually looked at this PR when it was made and wanted to say the same thing then, I just didn't actually make a review at the time

> # Switching between Pretendo Network and Nintendo Network
> 
> Occasionally, you may want to switch between your Pretendo Network and your Nintendo Network profiles on your Wii U; fortunately, it is super simple to switch between the two.
> 
> While connected to Pretendo, in the HOME Menu, switch to your Nintendo Network profile. If prompted for a password, do not provide it (if you don't have auto sign-in enabled, you may not have the option to not provide it). You are now inside your Nintendo Network profile.
> 
> On your Wii U GamePad, press the buttons `L + Down + SELECT`. This action opens up the Aroma plugin menu, which should look something like this:
> 
> <img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">
> 
> You may have additional plugins in your menu. Navigate to the `Inkay` plugin and press `A` to enter it's settings.
> 
> You should now be greeted with a menu displaying two buttons. Click the one that says 'Patching'.
> 
> To use Nintendo's servers, set 'Connect to the Pretendo Network' to **false**. It should look something like this:
> 
> <img src="/assets/images/docs/install/wiiu/pnid/inkay-patching.jpg" width="100%">
> 
> Once you've done that, press the `B` button twice, then press the `HOME` button. Your console will reboot with Pretendo Network patches disabled. You can check this by verifying whether a message saying 'Using Nintendo Network' is displayed on boot.
> <img src="/assets/images/docs/install/wiiu/pnid/nintendo-connection.jpg" width="100%">
> 
> After that, you should be able to access Nintendo Network. To go back to Pretendo, you can follow these steps again, this time signing into your Pretendo profile and ensuring that 'Connect to the Pretendo Network' is set to **true**.

Reviewer: A de-fluffed version could be this

```suggestion
# Switching between Pretendo Network and Nintendo Network

To switch between your Pretendo Network and Nintendo Network profiles on your Wii U:

Press `L + Down + SELECT` on the Wii U GamePad to open the Aroma plugin menu.

<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">

Locate and select 'Inkay', then choose 'Patching'. Now, set 'Connect to the Pretendo Network' to **false**.

<img src="/assets/images/docs/install/wiiu/pnid/inkay-patching.jpg" width="100%">

Press `B` twice, followed by the `HOME` button. The console will reboot with Pretendo Network patches disabled, as confirmed by the 'Using Nintendo Network' message.

Now, switch to your desired Network ID via the Account selection screen.

To return to Pretendo Network, repeat the process ensuring 'Connect to the Pretendo Network' is set to **true**.
```

Reviewer: @jonbarrow

Reviewer 2: lgtm
